include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.cpp

```diff
@@ -1030,7 +1135,9 @@ SpatialOperatorBase<dim, Number>::compute_vorticity(VectorType & dst, VectorType
 {
   vorticity_calculator.compute_vorticity(dst, src);
 
-  inverse_mass_velocity.apply(dst, dst);
+  // If HDIV is used, an iterative solver is used,
+  // but here we ignore the number of iterations
```

Comment:
I would not comment on HDIV here. Then, we would have to do this whenever we call the function.